blog/2023-04-openAPI-specification/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "OpenAPI Specifications"
+path: "/blog/2023-04-openAPI-specifications"
+date: "2023-04-07T11:15:00"
+author: minajanicijevic
+---
+
+
+# Unleashing the Power of OpenAPI Specifications: TriplyDB's Step Forward in RESTful API Development
```

Comment:
Either change the title above to this one or make this title a subtitle. Currently there are two different titles for this blog post.